lte/gateway/fabfile.py

```diff
@@ -536,7 +536,20 @@ def run_integ_tests(tests=None, federated_mode=False):
     if tests:
         tests = "TESTS=" + tests
 
-    execute(_run_integ_tests, gateway_ip, tests, federated_mode)
+    execute(_run_integ_tests, gateway_ip, tests, _to_boolean(federated_mode))
+
+
+def _to_boolean(value):
+    """Fabric function arguments passed by calling fab from external are strings.
```

Comment:
We are already using the following pattern throughout the fabfile:

```
    destroy_vm = bool(strtobool(destroy_vm))
    provision_vm = bool(strtobool(provision_vm))
```
see e.g. line 494-495.

I think this would also do for the variables we are fixing in this PR. Otherwise, there does not seem to be a reason not to just use `_to_boolean` for every instance of a string representing a bool in this file.

Reply:
This only works here because the default is already a string - e.g., `destroy_vm='True'`. This is not consistent in this fabfile and should be aligned. But I see this out of scope for a bug/fix PR.